a71.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Haky86",
+			"oem": "Samsung",
+			"device": "Galaxy A71 4G",
+			"filename": "crDroidAndroid-13.0-20231203-a71-v9.11.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/a71/9.x/crDroidAndroid-13.0-20231203-a71-v9.11.zip/download",
+			"timestamp": 1701626261,
+			"md5": "4b7a1d94671700393fa2529a6c3e38a7",
+			"sha256": "fe2baa378de80c7cbed8e0518ea112595a2b65b8384dbd1b0f96a3f5e499782b",
+			"size": 1291468398,
+			"version": "9.11",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-13-official-20231203-crdroid-9-11-for-samsung-galaxy-a71-4g-ss-ds.4644122/",
+			"gapps": "https://wiki.lineageos.org/gapps",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/haky86/files/SamsungGalaxyA71/Recoveries/crDroid/13.x/crDroidAndroid-13.0-20231203-a71-v9.11.tar/download",
+			"paypal": "https://paypal.me/Haky86",
+			"telegram": "https://t.me/GalaxyA714G",
+			"dt": "https://github.com/crdroidandroid/android_device_samsung_a71",
```

Comment:
Specifying 13.0 branch would be better :)
https://github.com/crdroidandroid/android_device_samsung_a71/tree/13.0